<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDbSql</class>
 <widget class="QWidget" name="frmDbSql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="gboxSqlite">
        <property name="title">
         <string>sqlite数据库</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labSqliteDBName">
           <property name="text">
            <string>数据库名称</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtSqliteDBName"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnSqliteConnect">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>连接测试</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labSqliteSql">
           <property name="text">
            <string>数据库脚本</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtSqliteSql"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnSqliteSql">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>执行</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxMysql">
        <property name="title">
         <string>mysql数据库</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labMysqlDBIP">
           <property name="text">
            <string>主机名称:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labMysqlUserName">
           <property name="text">
            <string>用户名称:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtMysqlDBName"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labMysqlDBName">
           <property name="text">
            <string>数据库名称:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtMysqlDBIP"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labMysqlDBPort">
           <property name="text">
            <string>连接端口:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtMysqlSql"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtMysqlUserName"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="txtMysqlUserPwd">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtMysqlDBPort"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labMysqlUserPwd">
           <property name="text">
            <string>用户密码:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labMysqlSql">
           <property name="text">
            <string>数据库脚本</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="btnMysqlSql">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>执行</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="btnMysqlConnect">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>连接测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
